04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -410,14 +816,410 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this case we queried an SDSS DR7 table at the TAP service run by GAVO in Heidelberg.  \n",
-    "\n",
-    "We can get the details of a particular profile by including the name of the profile as an argument in the <i>list_profiles</i> method:"
+    "In this case we queried an SDSS DR7 table at the TAP service run by GAVO in Heidelberg. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's ensure it has been saved to VOSpace:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "003918.fits,6_drz.fits,QuickResults.csv,cutout.fits,gavo_out.csv,newmags.csv,ps1.vot,public,results,tmp\n"
+     ]
+    }
+   ],
+   "source": [
+    "listing = sc.ls (name='vos://')\n",
+    "print (listing)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can view the data table on-screen by converting it to a pandas dataframe using the *convert* function:"
```

Comment:
I wouldn't make the case that to view the data on screen one can convert to Pandas. It's more likley user wants to work with the data. Suggest "We can load the data set and e.g. convert to a Pandas data frame, like this:"